AppCenterCrashes/AppCenterCrashesTests/MSCrashesTestUtil.m

```diff
@@ -24,21 +24,23 @@ + (BOOL)createTempDirectory:(NSString *)directory {
 + (BOOL)copyFixtureCrashReportWithFileName:(NSString *)filename {
   NSFileManager *fm = [[NSFileManager alloc] init];
 
-  // the bundle identifier when running with unit tests is "otest"
-  const char *progName = getprogname();
-  if (progName == NULL) {
-    return NO;
+  NSString *bundleIdentifier = [[NSBundle mainBundle] bundleIdentifier];
+  if (!bundleIdentifier) {
```

Comment:
Can you just add a brief comment that explains this block?

Reply:
It's just a logic from PLCR -https://github.com/microsoft/plcrashreporter/blob/050e1641de2b2d6ac157382a0123b8ed948941ac/Source/PLCrashReporter.m#L885

We didn't handle all the cases here before